### PR TITLE
[WIP] Adds basic user authentication and gulp

### PR DESCRIPTION
**Pull request by atjonathan**

### CODE REVIEW

**Comment by alayek** [comment]
It's very late here right now. I will check in the morning!

File: app.js [hunk 1]
```diff
@@ -13,21 +21,68 @@ MongoDB.once('open', function() {
...
+app.use(passport.initialize());
+app.use(passport.session());
+
+passport.use(new GitHubStrategy({
+    clientID: config.get('integrations.github.clientID'),
+    clientSecret: config.get('integrations.github.clientSecret'),
+    callbackURL: "http://127.0.0.1:3002/auth/github/callback"
+  },
+  function(accessToken, refreshToken, profile, cb) {
+    console.log(profile);
```
Comment (atjonathan) [inline app.js]: Remove this line

**Comment by alayek** [comment]
I will test this out ASAP.

**Comment by atjonathan** [comment]
@alayek, hold up there are some bugs. I need to implement some es6 stuff still. I'll let you know when it's ready :smile:

**Comment by atjonathan** [comment]
@alayek, try now...

**Pull request merged**